Formula/scotch.rb

```diff
@@ -29,25 +29,26 @@ class Scotch < Formula
   def install
     cd "src"
     (buildpath/"src").install_symlink "Make.inc/Makefile.inc.i686_mac_darwin10" => "Makefile.inc"
-    system "make"
+
+    inreplace "Makefile.inc" do |s|
+      s.change_make_var! "CCS", ENV.cc
+      s.change_make_var! "CCP", "mpicc"
+      s.change_make_var! "CCD", "mpicc"
+    end
+
+    system "make", "scotch", "ptscotch"
     system "make", "prefix=#{prefix}", "install"
+
+    pkgshare.install "check/test_strat_seq.c"
+    pkgshare.install "check/test_strat_par.c"
   end
 
   test do
-    (testpath/"test.c").write <<~EOS
-      #include <stdlib.h>
-      #include <stdio.h>
-
-      #include <scotch.h>
-      int main(void) {
-        int major, minor, patch;
-        SCOTCH_version(&major, &minor, &patch);
-        printf("%d.%d.%d", major, minor, patch);
-        return 0;
-      }
-    EOS
-
-    system ENV.cc, "test.c", "-L#{lib}", "-lscotch"
-    assert_match version.to_s, shell_output("./a.out")
```

Comment:
We don't need to remove this.

Reply:
Fine. I'll keep this one for scotch testing.